xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.h

```diff
@@ -94,6 +95,19 @@ namespace OVERLAY {
     float m_height;
     float m_source_width{0}; // Video source width resolution used to calculate aspect ratio
     float m_source_height{0}; // Video source height resolution used to calculate aspect ratio
+
+  protected:
+    /*!
+     * \brief Given the resolution ratio determines if it is a 4/3 resolution
+     * \param resRatio The resolution ratio (the results of width / height)
+     * \return True if the ratio refer to a 4/3 resolution, otherwise false
+     */
+    bool IsSquareResolution(float resRatio)
+    {
+      // 1:25 is the ratio of old resolutions like PAL (e.g. 720x576)
+      // 1:33 is the ratio of the most common 4/3 resolutions
+      return std::fabs(resRatio - 1.25f) < 0.001f || std::fabs(resRatio - 1.33f) < 0.001f;
```

Comment:
Do we want it to return true with 640x480 or 768x576 resolutions?

i.e:  `IsSquareResolution(640.0f / 480.0f)` 

I'm also not sure why only these two AR's are tested. Perhaps it would be better to compare with a larger interval to include possible intermediate AR:

`return (resRatio > 1.22f && resRatio < 1.34f)`

Perhaps this fits more in "IsSquareResolution"

Reply:
the two ratios is because is used for DVB subtitle resolution case, not for all video resolutions
maybe i should have given a different name to method

for the dvb subtitle this is correct, we are talking of a use case of DVB subtitle with SD resolution (the two ratios) on a 16/9 res tv channel (as in the code comment),
if this not match is because customised resolutions have been used which should not be tv channels, but more likely you are playing a bad encoded video file

in short, i can extend the range and keep same method name, will probably should have an effect on weirdly encoded video files
IMO these 4/3 contents with uncommon resolutions should not be frequent

Reply:
change applied